src/Test/FakeTemplating.php

```diff
@@ -20,6 +20,8 @@
  * Mocking class for template usage.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @deprecated since 4.0, to be removed with 5.0.
  */
 class FakeTemplating implements EngineInterface
```

Comment:
This class is not used any more. Can i remove it instead of deprecating?

Reply:
Yes